src/components/MultiHopTrade/hooks/useGetTradeQuotes.tsx

```diff
@@ -84,67 +67,5 @@ export const useGetTradeQuotes = () => {
     wallet,
   ])
 
-  const lifiQuery = useGetLifiTradeQuoteQuery(debouncedTradeQuoteInput, {
-    skip: !flags.LifiSwap,
-  })
-
-  const thorQuery = useGetThorTradeQuoteQuery(debouncedTradeQuoteInput, {
-    skip: !flags.ThorSwap,
-  })
-
-  // TODO(woodenfurniture): quote selection
-  const sortedQuotes: TradeQuoteResult[] = useMemo(() => {
-    if (isSkipToken(debouncedTradeQuoteInput)) return []
-
-    const results = [
-      {
-        isLoading: thorQuery.isFetching,
-        data: thorQuery.data,
-        error: thorQuery.error,
-        swapperName: SwapperName.Thorchain,
-      },
-      {
-        isLoading: lifiQuery.isFetching,
-        data: lifiQuery.data,
-        error: lifiQuery.error,
-        swapperName: SwapperName.LIFI,
-      },
-    ]
-      .filter(result => {
-        const swapperSupportsCrossAccountTrade = isCrossAccountTradeSupported(result.swapperName)
-        return !isCrossAccountTrade || swapperSupportsCrossAccountTrade
-      })
-      .map(result => {
-        const quote = result.data && result.data.isOk() ? result.data.unwrap() : undefined
-        const inputOutputRatio = quote
-          ? getInputOutputRatioFromQuote({
-              quote,
-              swapperName: result.swapperName,
-            })
-          : -Infinity
-        return Object.assign(result, { inputOutputRatio })
-      })
-
-    return orderBy(results, ['inputOutputRatio', 'swapperName'], ['asc', 'asc'])
-  }, [
-    debouncedTradeQuoteInput,
-    thorQuery.isFetching,
-    thorQuery.data,
-    thorQuery.error,
-    lifiQuery.isFetching,
-    lifiQuery.data,
-    lifiQuery.error,
-    isCrossAccountTrade,
-  ])
-
-  const bestQuote: TradeQuoteResult = sortedQuotes[0]
-  const quote = bestQuote?.data && bestQuote.data.isOk() ? bestQuote.data.unwrap() : undefined
-  const error = bestQuote?.data && bestQuote.data.isErr() ? bestQuote.data.unwrapErr() : undefined
-  dispatch(tradeQuoteSlice.actions.setSwapperName(bestQuote?.swapperName))
-  dispatch(tradeQuoteSlice.actions.setQuote(quote))
-  dispatch(tradeQuoteSlice.actions.setError(error))
-  return {
-    sortedQuotes,
-    selectedQuote: sortedQuotes[0],
-  }
+  useGetTradeQuoteQuery(debouncedTradeQuoteInput, { pollingInterval: 10000 })
```

Comment:
10 seconds for now, though subject to change.

Reply:
might be worth setting `keepUnusedDataFor` here in case a change to the polling interval causes the cache to invalidate between polling

Reply:
not blocking, but i just realised that there are 3 places this hook will be instantiated, so we should think about cancelling requests on unmount

Reply:
I reckon that's a non-issue, polling will be automatically cancelled on unmount, but queries will persist for 60 seconds (default `keepUnusedDataFor`) after the last subscriber unsubscribed to the query. I.e when all consumers are eventually unmounted